docs/tfengine/schemas/resources.md

```diff
@@ -233,3 +238,10 @@
 | storage_buckets.retention_policy.retention_period | The period of time, in seconds, that objects in the bucket must be retained and cannot be deleted, overwritten, or archived. The value must be less than 2,147,483,647 seconds. | number | false | - | - |
 | storage_buckets.storage_location | Location to create the storage bucket. Can be defined in global data block. | string | false | - | - |
 | terraform_addons | Additional Terraform configuration for the project deployment. Can be used to support arbitrary resources not supported in the following list. For schema see ./deployment.hcl. | object | false | - | - |
+| workload_identity_configurations | [Module](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/tree/master/modules/workload-identity) | array() | false | - | - |
```

Comment:
Consider dropping "_configurations" and just have "workload_identity"

Reply:
Done.